### PR TITLE
proofreading fixes

### PR DESCRIPTION
**Pull request by wstephenson**

### CODE REVIEW

File: PACKAGE.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ osc sr systemsmanagement:SCC:RMT rmt-server openSUSE:Factory
 
 ##### Submit maintenance updates for SLES to the Internal Build Service
 
-###### Get target codesteams where to submit
+###### Get target codestreams where to submit
```
Comment (ikapelyukhin) [inline PACKAGE.md]: `PACKAGING.md` *cough-cough*

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: PACKAGE.md (hunk 1 above)
Comment (lagartoflojo) [reply PACKAGE.md]: Which two places?

Comment (ikapelyukhin) [reply PACKAGE.md]: @lagartoflojo https://github.com/SUSE/rmt/blob/master/PACKAGE.md

Comment (ikapelyukhin) [reply PACKAGE.md]: @lagartoflojo arr, nevermind! I thought that was in `README.md` :grimacing:

**Pull request merged**